src/kit/Typography.module.scss

```diff
@@ -0,0 +1,97 @@
+/* stylelint-disable custom-property-pattern */
+
+:root {
+  /* Title */
+
+  /* font-size */
+  --title-size-l: 24px;
+
+  /* line-height */
+  --title-ln-l: 32px;
+  --title-size-default: 22px;
+  --title-ln-default: 28px;
+  --title-size-s: 18px;
```

Comment:
nit: move the sizes out of the line height section

Reply:
this applies to body and label as well

Reply:
changing this to just group styles by title/body/label type